client-go/controller/models/ps/ps.go

```diff
@@ -0,0 +1,98 @@
+package ps
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/deis/deis/client-go/controller/api"
+	"github.com/deis/deis/client-go/controller/client"
+)
+
+// List an app's processes.
+func List(c *client.Client, appID string) ([]api.Process, error) {
+	u := fmt.Sprintf("/v1/apps/%s/containers/", appID)
+	body, status, err := c.BasicRequest("GET", u, nil)
+
+	if err != nil {
+		return []api.Process{}, err
+	}
+
+	if status != 200 {
+		return []api.Process{}, errors.New(body)
+	}
+
+	processes := api.Processes{}
```

Comment:
Maybe the local variable should be `procs`? I had to read the `processes.Processes` line a couple times before it parsed correctly.